src/map/atcommand.cpp

```diff
@@ -7823,15 +7852,29 @@ ACMD_FUNC(mobinfo)
 				if (mvppercent > 0) {
 					j++;
 					if (j == 1) {
-						if (id->slots)
-							sprintf(atcmd_output2, " %s[%d]  %02.02f%%", id->ename.c_str(), id->slots, mvppercent);
-						else
-							sprintf(atcmd_output2, " %s  %02.02f%%", id->ename.c_str(), mvppercent);
+						if (id->slots){
+							if (!battle_config.feature_itemlink)
+								sprintf(atcmd_output2, " %s[%d] %02.02f%%", id->ename.c_str(), id->slots, mvppercent);
+							else
+								sprintf(atcmd_output2, " %s %02.02f%%", createItemLink(tmp_item).c_str(), mvppercent);
+						} else {
+								if (!battle_config.feature_itemlink)
+									sprintf(atcmd_output2, " ⁕ %s %02.02f%%", id->ename.c_str(), mvppercent);
+								else
+									sprintf(atcmd_output2, " ⁕ %s %02.02f%%", createItemLink(tmp_item).c_str(), mvppercent);
+						}
 					} else {
-						if (id->slots)
-							sprintf(atcmd_output2, " - %s[%d]  %02.02f%%", id->ename.c_str(), id->slots, mvppercent);
-						else
-							sprintf(atcmd_output2, " - %s  %02.02f%%", id->ename.c_str(), mvppercent);
+							if (id->slots){
+								if (!battle_config.feature_itemlink)
+									sprintf(atcmd_output2, " %s[%d] %02.02f%%", id->ename.c_str(), id->slots, mvppercent);
+								else
+									sprintf(atcmd_output2, " %s %02.02f%%", createItemLink(tmp_item).c_str(), mvppercent);							
+							} else {
+									if (!battle_config.feature_itemlink)
+										sprintf(atcmd_output2, " ⁕ %s %02.02f%%", id->ename.c_str(), mvppercent);
+									else
+										sprintf(atcmd_output2, " ⁕ %s %02.02f%%", createItemLink(tmp_item).c_str(), mvppercent);
+							}
```

Comment:
(just another extra indentation)
```suggestion
						if (id->slots){
							if (!battle_config.feature_itemlink)
								sprintf(atcmd_output2, " %s[%d] %02.02f%%", id->ename.c_str(), id->slots, mvppercent);
							else
								sprintf(atcmd_output2, " %s %02.02f%%", createItemLink(tmp_item).c_str(), mvppercent);							
						} else {
							if (!battle_config.feature_itemlink)
								sprintf(atcmd_output2, " ⁕ %s %02.02f%%", id->ename.c_str(), mvppercent);
							else
								sprintf(atcmd_output2, " ⁕ %s %02.02f%%", createItemLink(tmp_item).c_str(), mvppercent);
						}
```